samples/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
+cmake_minimum_required(VERSION 3.4.3)
```

Comment:
This file seems to be missing the license text, could you please add that as well?